hackvote/app/controllers/events_controller.rb

```diff
@@ -0,0 +1,41 @@
+class EventsController < ApplicationController
+
+  NUMBER_OF_VOTES = 3
+
+  def new
+	  @event = Event.new
+	  @cur_events = Event.where( completed: false)
+	  @past_events = Event.where( completed: true)
+  end
```

Comment:
Indentation is jacked; use 2 space indentation for ruby. Probably should setup your text editor to make tabs insert as 2 spaces for ruby files.